server/service/src/sync/program_schemas/hiv_care_encounter.json

```diff
@@ -256,26 +242,35 @@
           "properties": {
             "bloodPressure": {
               "description": "75367002\tBlood pressure [mmHg]",
-              "type": "number"
+              "pattern": "^\\d{3}\\/\\d{2}$",
+              "title": "BP in mmHg",
+              "type": "string",
+              "messages": { "pattern": "Format is 000/00" }
             },
             "bodyMassIndex": {
               "description": "60621009\tBody mass index (BMI)",
               "type": "number"
             },
             "height": {
               "description": "50373000\tBody height measure [m]",
-              "type": "number"
+              "pattern": "^\\d\\.\\d{2}$",
+              "title": "Height in meters",
+              "type": "string",
+              "messages": { "pattern": "Format is 0.00" }
             },
             "midUpperArmCircumference": {
               "description": "284473002\tMid upper arm circumference (MUAC) [cm]",
-              "type": "number"
+              "pattern": "^\\d+\\.\\d{2}$",
+              "title": "MUAC in cm",
+              "type": "string",
+              "messages": { "pattern": "Format is 0.00" }
             },
             "nutritionalStatus": {
               "description": "366364004  nutritional status",
               "enum": ["Under nourished", "Well nourished", "Overweight"],
               "type": "string"
             },
-            "opportunisticInfection": {
+            "opportunisticInfections": {
```

Comment:
do we need to update this somewhere else ?

Reply:
it was updated in the programschema repo and in the uischema. was just missed from this one.